### ISSUE TITLE
Move to some org?

### ISSUE THREAD

**Issue by DanielRuf** [terser/html-minifier-terser#15]

**Comment by shellscape** [comment]
Please don't 😄

**Comment by DanielRuf** [comment]
> Please don't 😄

Hi @shellscape,

please clarify why you think that this is not a good idea.

I am webpack contributor and I think it's better this project should be under an org than my personal profile. It could be also another org.

Where should I move this to?

**Comment by shellscape** [comment]
I was a core webpack member in past years, now on the Rollup team. We're evaluating this package for a new plugin in discussion. There's absolutely zero bad blood between the two projects but we have seen maintenance slow on projects moved to the webpack org, simply because it's so vast. I'm sure there are varying opinions, but that's my take.

If I were to make a recommendation on an org, I would choose https://github.com/terser.

**Comment by DanielRuf** [comment]
cc @fabiosantoscode what do you think? How can we ensure a bigger bus factor and that it will be maintained for a long time?

**Comment by DanielRuf** [comment]
Thanks for your explanation and feedback.
Let's discuss this further.

@fabiosantoscode what do you think? How can we ensure a bigger bus factor and that it will be maintained for a long time?

**Comment by fabiosantoscode** [comment]
I don't know. I'm not exactly an open source specialist :) Terser has its own org but it's really just me. There are people who can transfer ownership of the Terser repository in case I get hit by a bus, so I haven't really given any thought to a bigger organisation.

**Comment by rodrigograca31** [comment]
My 2 cents:

Even if its just @fabiosantoscode I think it would still be a good move.
Because for example the other day @DanielRuf "disappeared" from Keybase and I was like "Is he ok? what will happen to his project?"

@fabiosantoscode maybe you could add @DanielRuf along with this project?

**Comment by DanielRuf** [comment]
I'm not sure if there will be some action from https://github.com/DanielRuf/html-minifier-terser/issues/46#issuecomment-668517987.

I'm not further discussing this but I plan to create a separate org to maintain it if needed but would welcome the move to the terser org as this makes much more sense from my view / imho.

**Comment by fabiosantoscode** [comment]
Yeah, let's do that. The increased bus factor on both projects should be welcome.

I'm sending you an invite @DanielRuf

**Comment by rodrigograca31** [comment]
Yay! glad I could "help" 😅

Also just noticed @fabiosantoscode is a Portuguese fellow :smile:

**Comment by fabiosantoscode** [comment]
Haha obrigado pela sugestão :)

**Comment by DanielRuf** [comment]
I have accepted the invitation to the terser organization.
The repo is now at https://github.com/terser/html-minifier-terser and the increased busfactor should reduce some risks.

There will be no change regarding my activity regarding this project in the near future.
I will send some npmjs invites in the next weeks after asking who wants to join as maintainer there as backup.

I will likely provide some of my resources (my time) to the other projects in the terser org too and try to help a bit with the communities around terser as I see this as something where I can give back and support the others.

Closing as the move is completed.
Thanks for the help with this.
